the22mastermind/wfa
# Add weather forecast component, optimized dashboard component, and se…

#### What does this PR do?
1. Add weather forecast component
2. Refactor dashboard component
3. Secured API key.
#### Description of Task to be completed?
1. Create weather forecast component
2. Refactor dashboard component
3. Use .env file to store API key
#### How should this be manually tested?
1. Open your terminal and run `git clone https://github.com/the22mastermind/wfa.git`
2. run `cd wfa`
3. run `npm install` to install the dependencies
4. run `npm start`
5. Navigate to sign in and enter any email and password
6. Enter the name of any city in the search box
7. Hit `View forecast` button to see weather forecasts up to 5 days
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
#### Questions:
